dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1252,8 +1252,7 @@ public WesRequestData aggregateWesRequestData(final WesCommandParser wesCommandP
 
                 // Build and return the request data
                 return new WesRequestData(wesEndpointUrl,
-                    credentialsProvider.getCredentials().getAWSAccessKeyId(),
```

Comment:
Formatting/width is a bit funny here, eventually we'll want to sync checkstyle? between repos